package.json

```diff
@@ -13,27 +13,23 @@
     "dist"
   ],
   "devDependencies": {
-    "bluebird": "^3.5.1",
-    "construct-style-sheets-polyfill": "^2.1.0",
+    "@rollup/plugin-replace": "^2.3.1",
+    "bluebird": "^3.7.2",
+    "construct-style-sheets-polyfill": "^2.3.5",
     "esm": "^3.2.25",
-    "mocha": "^5.2.0",
+    "mocha": "^7.1.1",
     "opn": "^6.0.0",
-    "rollup": "^0.64.1",
-    "rollup-plugin-replace": "^2.0.0",
-    "terser": "^3.8.1",
-    "whatwg-fetch": "^2.0.4"
+    "rimraf": "^3.0.2",
+    "rollup": "^2.2.0",
+    "rollup-plugin-terser": "^5.3.0",
+    "terser": "^4.6.7",
+    "whatwg-fetch": "^3.0.0"
   },
   "scripts": {
-    "build": "npm run build:systemjs && npm run build:sjs && npm run min && npm run build:extras",
-    "build:systemjs": "rollup -c",
-    "build:sjs": "rollup -c --environment sjs",
-    "build:extras": "bash minify-extras.sh",
-    "min": "npm run min:systemjs && npm run min:sjs",
-    "min:systemjs": "cd dist && terser system.js -c passes=2 -m --source-map --comments \"/SystemJS \\d/\" -o system.min.js",
-    "min:sjs": "cd dist && terser s.js -c passes=2 -m --source-map -o s.min.js",
+    "build": "rimraf dist && rollup -c",
```

Comment:
We should retain a build mode / option that allows skipping minification here for quick builds in development.

I seem to recall there may even be a way with RollupJS to pass options, but can't remember if that is correct. Otherwise environment variables can work too.

Reply:
You can with rollup's `--environment` flag